lib/config/definitions.ts

```diff
@@ -593,7 +593,7 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'npmToken',
-    description: 'npm token used for authenticating with the default registry',
+    description: 'NPM token used for authenticating with the default registry',
```

Comment:
```suggestion
    description: 'npm token used for authenticating with the default registry',
```

Revert because npm should always be in lower case